core/server/master/src/main/java/alluxio/master/file/options/CreatePathOptions.java

```diff
@@ -199,6 +215,25 @@ public T setMetadataLoad(boolean metadataLoad) {
     return getThis();
   }
 
+  /**
+   * @param ttl the TTL (time to live) value to use; it identifies duration (in milliseconds) the
+   *        created file should be kept around before it is automatically deleted
+   * @return the updated options object
+   */
+  public T setTtl(long ttl) {
+    getCommonOptions().setTtl(ttl);
+    return getThis();
+  }
+
+  /**
+   * @param ttlAction the {@link TtlAction}; It informs the action to take when Ttl is expired;
+   * @return the updated options object
+   */
+  public T setTtlAction(TtlAction ttlAction) {
+    getCommonOptions().setTtlAction(ttlAction);
+    return getThis();
+  }
+
   @Override
   public boolean equals(Object o) {
```

Comment:
update this method?

Reply:
Do you mean the equals() method?
Since Ttl and TtlAction are included in CommonOptions, this equals should be fine?

Reply:
ack